.github/workflows/repo-security-audit.yml

```
@@ -0,0 +1,405 @@
+          echo "🚨 SECURITY GATE FAILED"
+          echo "Critical Issues: ${{ env.CRITICAL_ISSUES }}"
+          echo "High Issues: ${{ env.HIGH_ISSUES }}"
+          echo "Security Score: ${{ env.SECURITY_SCORE }}/100"
+          echo ""
+          echo "❌ This PR introduces critical security vulnerabilities."
+          echo "Please review and fix security issues before merging."
+          exit 1
+          
+      - name: 📝 Generate Security Report
+        if: always()
+        run: |
```

actionlint (1.7.7)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)